.travis.yml

```diff
@@ -0,0 +1,27 @@
+before_script:
+  - ruby --version
+
+env:
+  global:
+    - APM_TEST_PACKAGES="language-haml linter"
```

Comment:
I had issues with atom-package-deps not installing language-haml, e.g. https://travis-ci.org/jschroeder9000/linter-haml/jobs/115568631.